src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/service/ItemGroupsService.java

```diff
@@ -34,6 +47,8 @@ public ItemGroup createItemGroup(ItemGroupData itemGroupData) {
         setCreationTimeStamp(itemGroup);
         itemGroup.setData(itemGroupData);
 
+        regenerateLinks(itemGroupData, itemGroupId);
+
```

Comment:
I think in a way similar (but simpler) to the way we generate the `self` links here on persisting the item group, we will want to generate an `etag` too. This was recently added to the spec in [DCAC-53](https://github.com/companieshouse/private.api.ch.gov.uk-specifications/pull/158). Perhaps this could also be dealt with in a separate PR/ticket?

Reply:
Good idea 👍  @caismith-ch could this be included with the placeholder tickets?

Reply:
Yes definitely, will get it raised now

Reply:
Raised under `DCAC-249`